server/db/migrations/19_add_indexes_to_container_log.rb

```diff
@@ -1,8 +1,6 @@
 class AddIndexesToContainerLog < Mongodb::Migration
   def self.up
-    Thread.new {
-      Thread.current.abort_on_exception = true
-      ContainerLog.create_indexes # might take a long time
-    }
+    info "recreating container log indexes, this might take long time"
+    ContainerLog.create_indexes
```

Comment:
If master is deployed with some rolling-upgrade mechanism then background index will help.